articles/TOC.yml

```diff
@@ -1,44 +1,51 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Getting started
   items:
-  - name: Get started
-    href: welcome.md
-  - name: Five questions about quantum computing
-    items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: The QDK introduction
+  href: welcome.md
```

Comment:
Made further rearrangement and changed titles in the TOC. But this particular item always repeats with the parent title. It should be "Get started with the Quantum Development Kit (QDK)" as in the welcome.md. Not sure how to change this child title.

![image](https://user-images.githubusercontent.com/59890648/73982804-d537b880-48e9-11ea-96a5-afb326a70229.png)

Reply:
Ah yes, I know the struggle--my first few months consisted of a lot of trial-and-error with these TOC things. It looks like that was coming from the fact that the page file `href: welcome.md` was directly under the `name: ...`, which then made the page automatically inherit the `name`. Got it fixed in a new commit (not yet pushed as I'm working through these comments)

Reply:
@KittyYeungQ one important thing though: with this change, I think we definitely need to change the title of the "Getting started" dropdown in the TOC. Not only would that be confusing to navigate, but with this page no longer in there, "Getting started" isn't really that descriptive (it's just the "five questions" in there)